.gitignore

```diff
@@ -1,19 +0,0 @@
-# Thumbnails
```

Comment:
Can we add the .gitignore back in? Nice to make sure we keep system files from various dev environments out of the repo. The .DS_Store files for example.

Reply:
That should go back in, yes.